---
title: "Sanders Lab - Software"
layout: textlay
excerpt: "Sanders Lab -- Software."
sitemap: false
permalink: /software/
---


# Software

---

## [scGNN](https://github.com/juexinwang/scGNN)


<!-- <img src="{{ site.url }}{{ site.baseurl }}/images/codepic/WGS_Pipeline_Image.png" style="width: 300px">  -->

scGNN is an iterative approach using three multi-modal autoencoders for gene imputation and cell clustering using graph neural networks integrated with LTMG model of gene expression in single cell RNA-seq analysis.

- <a href="https://github.com/juexinwang/scGNN"><i class='fab fa-github'></i> Source</a>
- <a href="https://www.nature.com/articles/s41467-021-22197-x"><i class='fa fa-book'></i> Paper</a>

---

## [NRI-MD](https://github.com/juexinwang/NRI-MD)


<!-- <img src="{{ site.url }}{{ site.baseurl }}/images/codepic/magellanmapper.png" style="width: 300px">  -->

Based on a graph neural network, NRI-MD is a neural relational inference model adopts an encoder-decoder architecture to simultaneously infer latent interactions for probing protein allosteric processes as dynamic networks of interacting residues. From the MD trajectories, this model successfully learned the long-range interactions and pathways that can mediate the allosteric communications. Furthermore, the model can discover allostery-related interactions earlier in the MD simulation trajectories and predict relative free energy changes upon mutations more accurately than other methods..

- <a href="https://github.com/juexinwang/NRI-MD"><i class='fab fa-github'></i> Source</a>
- <a href="https://www.nature.com/articles/s41467-022-29331-3"><i class='fa fa-book'></i> Paper</a>
<!-- - <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/abs/10.1002/cpns.104"><i class='fa fa-book'></i> Protocol</a> -->

---

## [GRGNN](https://github.com/juexinwang/GRGNN)


<!-- <img src="{{ site.url }}{{ site.baseurl }}/images/codepic/wgspower.png" style="width: 300px">  -->

GRGNN is an end-to-end approach to reconstruct gene regulatory networks from scratch utilizing the gene expression data, in both a supervised and a semi-supervised framework..

- <a href="https://github.com/juexinwang/GRGNN"><i class='fab fa-github'></i> Source</a>
- <a href="https://pubmed.ncbi.nlm.nih.gov/33294129/"><i class='fa fa-book'></i> Paper</a>

---

## [G2S](https://g2s.genomenexus.org/)

<!-- <img src="{{ site.url }}{{ site.baseurl }}/images/codepic/scn2aviz.png" style="width: 300px">  -->

An auto-updating web-service to annotate genomic variants on protein structures. Since 2016, G2S is official API supports structure mapping services in cBioPortal for cancer genomics.

- <a href="https://github.com/genome-nexus/g2s"><i class='fa fa-link'></i> Source</a>
- <a href="https://g2s.genomenexus.org/"><i class='fa fa-link'></i> Link</a>
- <a href="https://academic.oup.com/bioinformatics/article/34/11/1949/4827683"><i class='fa fa-book'></i> Paper</a>

---


## [BHIT](https://sourceforge.net/projects/cnvision/)


<!-- <img src="{{ site.url }}{{ site.baseurl }}/images/codepic/cnvision.png" style="width: 300px">  -->

Bayesian High-order Interaction Toolkit for genotype-phenotype epistasis inference.

- <a href="http://digbio.missouri.edu/BHIT/"><i class='fa fa-link'></i> Source</a>
- <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2217-6"><i class='fa fa-book'></i> Paper</a>


---



